crates/volta-core/src/run_package_global/npx.rs

```diff
@@ -0,0 +1,48 @@
+use std::ffi::OsString;
+
+use super::executor::{ToolCommand, ToolKind};
+use super::{debug_active_image, debug_no_platform};
+use crate::error::{ErrorKind, Fallible};
+use crate::platform::{Platform, System};
+use crate::session::Session;
+use crate::version::parse_version;
+
+/// Build a `ToolCommand` for npx
+pub(super) fn command(args: &[OsString], session: &mut Session) -> Fallible<ToolCommand> {
+    let platform = Platform::current(session)?;
+
+    Ok(ToolCommand::new("npx", args, platform, ToolKind::Npx))
+}
+
+/// Determine the execution context (PATH and failure error message) for npx
+pub(super) fn execution_context(
+    platform: Option<Platform>,
+    session: &mut Session,
+) -> Fallible<(OsString, ErrorKind)> {
+    match platform {
+        Some(plat) => {
+            let image = plat.checkout(session)?;
+
+            // npx was only included with npm 5.2.0 and higher. If the npm version is lower
+            // that that, we should include a helpful error message
+            let required_npm = parse_version("5.2.0")?;
```

Comment:
Follow-up per our discussion: maybe make this a `lazy_static!` that panics (presumably via `expect`) if it's done incorrectly, since users will never be able to do *anything* if it is wrong.

Reply:
Moved it to a `lazy_static` and actually used the infallible `Version::new` constructor, since we know the values for the version parts.